src/components/deviceprofileBuilder.ts

```diff
@@ -41,11 +41,11 @@ let profileOptions: ProfileOptions;
 let currentDeviceId: number;
 
 /**
- * @param Property What property the condition should test.
- * @param Condition The condition to test the values for.
- * @param Value The value to compare against.
- * @param [IsRequired=false]
```

Comment:
Why the default value is removed?

Reply:
I didn't understand how this would fit in to the jsdoc syntax, from the other functions I didn't see any practice of documenting default values with this syntax. Feel free to suggest a change

Reply:
Okay, fixed this :)